locale/en/blog/release/v0.10.42.md

```diff
@@ -7,6 +7,10 @@ slug: node-v0-10-42
 layout: blog-post.hbs
 author: James M Snell
 ---
+<!--lint disable prohibited-strings-->
```

Comment:
Can you confirm such comments are not in the built HTML? Not a blocker, but we should be aware about it.

Reply:
Yes, it looks like it ends up the built HTML